RealmModelGenerator/EntitieViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  EntitieViewController.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/21/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+
+class EntitiesViewController: NSViewController, EntitiesViewDelegate, EntitiesViewDataSource {
+    static let TAG = NSStringFromClass(EntitiesViewController)
+    
+    @IBOutlet weak var entitiesView: EntitiesView!
+    
+    private var schema:Schema = Schema()
+    private var model: Model = Schema.init().createModel()
+    private var entities:[Entity] = []
+    private var selectedIndex = -1
+    
+    override func viewWillAppear() {
+        entitiesView.delegate = self
+        entitiesView.dataSource = self
+        
+        if let currentModel = schema.getCurrentModel() {
+            model = currentModel
+        } else {
+            model = schema.createModel()
+        }
+
+        entities = model.entities
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+    
+    override var representedObject: AnyObject? {
+        didSet {
+            // Update the view, if already loaded.
+        }
+    }
+    
+    //MARK: - EntitiesViewDelegate
+    func entitiesView(entitiesView: NSTableView, viewForTableColmn tableColumn: NSTableColumn?, row: Int) -> NSView? {
+        let cell = entitiesView.makeViewWithIdentifier(TitleCell.IDENTIFIER, owner: nil) as! TitleCell
+        cell.title = entities[row].name
+        return cell
+    }
+    
+    func entitiesViewSelectionDidChange(notification: NSNotification) {
```

Comment:
rename function to entitiesView(entitiesView:EntitiesView selectionDidChange index:Int)